lab_1/main.py

```diff
@@ -21,7 +34,23 @@ def remove_stop_words(tokens: list, stop_words: list) -> list or None:
     :param stop_words: a list of stop words
     :return: a list of tokens without stop words
     """
-    pass
+
+    if not isinstance(tokens, list) or not isinstance(stop_words, list):
+        return None
+
+    filt_tokens = []
+    for token in tokens:
+        if token not in stop_words:
+            filt_tokens.append(token)
+    return filt_tokens
+
+tokens_en = tokenize(en_text)
+tokens_de = tokenize(de_text)
+tokens_unknown = tokenize(unknown_text)
+
+stop_words_en = ['a', 'an', 'the', 'and']
+stop_words_de = ['der', 'das', 'die', 'ein', 'eine']
+stop_words_unknown = []
```

Comment:
The `main.py` file should not contain function calls - you should check your implementation using the `start.py` file.